include/openrave/robot.h

```diff
@@ -450,19 +510,44 @@ class OPENRAVE_API RobotBase : public KinBody
     class OPENRAVE_API AttachedSensorInfo
     {
 public:
-        AttachedSensorInfo() {
+        AttachedSensorInfo() {}
+        AttachedSensorInfo(const AttachedSensorInfo& other) {
+            *this = other;
+        };
+        AttachedSensorInfo& operator=(const AttachedSensorInfo& other) {
+            _id = other._id;
+            _name = other._name;
+            _linkname = other._linkname;
+            _trelative = other._trelative;
+            _sensorname = other._sensorname;
+            rapidjson::Document docSensorGeometry;
```

Comment:
@bucketzxm @ziyan is there a way to re-use the same allocator in `_docSensorGeometry` without having memory grow?